book/caching.md

```diff
@@ -0,0 +1,33 @@
+Caching
```

Comment:
Isn't it already covered well by official guide? https://github.com/yiisoft/yii2/blob/master/docs/guide/caching-data.md